dashboard/src/store/mockStore.js

```diff
@@ -0,0 +1,30 @@
+// import configureStore to create a mock store where we will dispatch our actions
+import configureStore from "redux-mock-store";
+// import thunk middle to make our action asynchronous
+import thunk from "redux-thunk";
+
+// initialise middlewares
+const middlewares = [thunk];
+// initialise MockStore which is only the configureStore method which take middlewares as its parameters
+
+const mockStore = configureStore(middlewares);
```

Comment:
I think the blank line at line 9 belongs between lines 7 & 8, so that the comment at line 8 is "more attached" to line 10.

Reply:
Got it!